nexus/src/db/mod.rs

```diff
@@ -17,23 +17,23 @@ pub mod datastore;
 mod error;
 mod explain;
 pub mod fixed_data;
-pub mod ipv6;
 pub mod lookup;
 mod pagination;
 mod pool;
 // This is marked public because the error types are used elsewhere, e.g., in
 // sagas.
 pub(crate) mod queries;
 mod saga_recovery;
-mod saga_types;
 mod sec_store;
 mod update_and_check;
 
 #[cfg(test)]
 mod test_utils;
 
-pub mod model;
-pub mod schema;
+// TODO-jkg
```

Comment:
?

Reply:
Ah, nice catch. This comment is attached to re-exports that allow the rest of `nexus` to build without realizing `nexus-db-model` is now a separate crate; I meant to go back and remove these re-exports and rewrite relevant imports. I'll do that shortly.

Reply:
On second glance, I think rewriting all the relevant imports is more trouble than it's worth (particularly due to the way some of the authz-macros generate code with full path names to types). I'll just remove this comment.